src/components/Home/ConnectionCard/ConnectionCard.tsx

```diff
@@ -1,126 +1,202 @@
-        {t("home.conn_details")}&nbsp;
-        <Tooltip
-          overlay={<div>{showAddress ? t("home.hide") : t("home.show")}</div>}
-          placement="top"
-        >
-          {showAddress ? (
-            <EyeOffIcon
-              onClick={toggleAddressHandler}
-              className="inline-block h-6 w-6 cursor-pointer align-top"
-            />
-          ) : (
-            <EyeIcon
-              onClick={toggleAddressHandler}
-              className="inline-block h-6 w-6 cursor-pointer align-top"
-            />
-          )}
-        </Tooltip>
-      </div>
-      <article className="flex flex-col overflow-hidden py-4">
-        <h6 className="text-sm text-gray-500 dark:text-gray-200">
-          {t("home.tor")}
-        </h6>
-        <div className="flex">
-          <a
-            className={`${
-              showAddress
-                ? "w-10/12 overflow-hidden overflow-ellipsis text-blue-400 underline"
-                : "text-blur w-10/12"
-            }`}
-            title={`${torAddress}`}
-            href={`//${torAddress}`}
-            target="_blank"
-            rel="noreferrer"
-          >
-            {showAddress ? torAddress : HIDDEN_TEXT}
-          </a>
-          <Tooltip
-            overlay={
-              <div>
-                {clippedTor ? t("wallet.copied") : t("wallet.copy_clipboard")}
-              </div>
-            }
-            placement="top"
-          >
-            <ClipboardIcon
-              className="inline-flex h-6 w-2/12 cursor-pointer justify-self-end"
-              onClick={copyTor}
-            />
-          </Tooltip>
+  const showModalHandler = () => {
+    setShowModal(true);
+  };
+
+  const closeModalHandler = () => {
+    setShowModal(false);
+  };
+
+  const qrCodeModal =
+    showModal &&
+    createPortal(
+      <ModalDialog close={closeModalHandler}>
+        <div className="my-5 flex flex-col items-center justify-center">
+          <QRCodeSVG value={nodeId} size={256} />
+          <div className="mt-10 mb-3 text-sm text-gray-500">{nodeId}</div>
         </div>
-      </article>
-      <article className="flex flex-col overflow-hidden py-4">
-        <h6 className="text-sm text-gray-500 dark:text-gray-200">
-          {t("home.ssh")}
-        </h6>
-        <div className="flex">
-          <a
-            className={`${
-              showAddress
-                ? "w-10/12 overflow-hidden overflow-ellipsis text-blue-400 underline"
-                : "text-blur w-10/12"
-            }`}
-            title={`${sshAddress}`}
-            href={`ssh://${sshAddress}`}
-            target="_blank"
-            rel="noreferrer"
-          >
-            {showAddress ? sshAddress : HIDDEN_TEXT}
-          </a>
+      </ModalDialog>,
+      MODAL_ROOT
+    );
+
+  return (
+    <>
+      {qrCodeModal}
+      <div className="bd-card w-full transition-colors lg:mr-2 lg:w-1/2">
+        <div className="flex items-center text-lg font-bold">
```

Comment:
nit: not a div?

Reply:
Not sure about that since there is also a tooltip attached to it.